js/scripts.js

```diff
@@ -0,0 +1,69 @@
+const hamburger = document.querySelector('.navbar');
+const wrapperMain = document.querySelector('.wrapper');
+const mobileMenuContainer = document.querySelector('.mobile-menu-wrapper');
+const mobileMenuItem = document.querySelectorAll('.mobile-menu-item');
+const closeButton = document.querySelector('.closeBtn');
+const icon = document.querySelector('.fa-bars');
+const iconTimes = document.querySelector('.fa-times');
+const homepage = document.querySelector('.container-homepage');
+
+//mobileMenuContainer.style.display = 'none';
+
+
+function mobileMenuDisplay(){
+
+  if(icon.classList.contains('fa-bars')){
+    icon.classList.replace('fa-bars', 'fa-times');
+    mobileMenuContainer.style.display = 'block';
+    homepage.style.filter = 'blur(2px)';
+  }
+  else {
+      mobileMenuContainer.style.display = 'none';
+      icon.classList.replace('fa-times', 'fa-bars');
+  }
+  //wrapperMain.style.display = 'none';
+  
+}
+
+hamburger.addEventListener('click', mobileMenuDisplay);
+closeButton.addEventListener('click', mobileMenuDisplay);
```

Comment:
- [ ] You have really tried implementing the mobile menu functionality 👍🏽, but currently, the function you added to toggle the mobile menu is not working. When I click the hamburger, the mobile menu doesn't display but rather I get an error in the console. Kindly debug the issue and fix it 👍🏽

![image](https://user-images.githubusercontent.com/63473584/190004212-d901bcbd-1e03-457c-b993-dba45fc10021.png)